sdk/tables/Azure.Data.Tables/src/autorest.md

```diff
@@ -6,7 +6,7 @@
 ``` yaml
 title: Azure.Data.Tables
 input-file:
-    - https://github.com/shurd/azure-rest-api-specs/tree/tablesSwagger/specification/cosmos-db/data-plane/Microsoft.TablesStorage/preview/2018-10-10/table.json
+    - https://github.com/shurd/azure-rest-api-specs/tree/tablesSwagger/specification/cosmos-db/data-plane/Microsoft.TablesStorage/preview/2019-02-02/table.json
```

Comment:
Can you please use a link to specific commit here?

Because we run generated code check on every PR it might start failing if the swagger is updated upstream.